test/check-application

```diff
@@ -77,7 +77,7 @@ class TestApplication(testlib.MachineCase):
         self.allow_journal_messages("/run.*/podman/podman: couldn't connect.*")
         self.allow_journal_messages(".*/run.*/podman/podman.*Connection reset by peer")
 
-        self.has_criu = self.machine.image not in ["debian-testing", "ubuntu-stable"]
+        self.has_criu = self.machine.image not in ["debian-testing", "ubuntu-stable", "fedora-32"]
```

Comment:
fedora-33 will get here on the next refresh as well, so then we should just flip the list.